examples/Microsoft.Spark.CSharp.Examples/Sql/MachineLearning/SentimentAnalysis.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.ML;
+using Microsoft.ML.Data;
+using Microsoft.Spark.Sql;
+
+namespace Microsoft.Spark.Examples.Sql.MachineLearning
+{
+    /// <summary>
+    /// Example of using ML.NET + .NET for Apache Spark
+    /// for sentiment analysis.
+    /// </summary>
+    internal sealed class SentimentAnalysis : IExample
```

Comment:
If we want to do away with `IExample` interface everywhere, we can have separate folders and `.csproj` for every example (creating separate executables for every example).  This way you can make your ML examples and README a bit simpler I think  You can include some folder  `Examples\MachineLearning\Sentiment` and in this folder you can have a subfolder `SentimentML.Model` with a `SentimentML.Model.csproj` along with its source code.  In the root folder you can have Sentiment.csproj and add a `ProjectReference` to the `SentimentML.Model.csproj`.   This same comment would also be relevant to the other ML and original SQL/Streaming examples.

Reply:
I believe we addressed this feedback through our collaboration yesterday. Thank you again for all your help!